presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -411,6 +419,15 @@ public PlanOptimizers(
                 ImmutableSet.of(
                         new CreatePartialTopN(),
                         new PushTopNThroughUnion())));
+        builder.add(new IterativeOptimizer(
```

Comment:
I'm not sure, but it seems that it can be moved here as a separate commit

Reply:
@arhimondr You are correct. Moved to `Add SpatialJoinNode` commit.